prow/scripts/cluster-integration/helpers/generate-ark-config.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+
+echo -e "Create a Secret for Ark"
```

Comment:
there is a helper called shout, you can use it instead of echo

Reply:
changed to shout!